README.md

```diff
@@ -16,3 +16,12 @@ Plug 'tbodt/deoplete-tabnine', { 'do': 'powershell.exe .\install.ps1' }
 
   [Deoplete]: https://github.com/Shougo/deoplete.nvim/
   [TabNine]: https://tabnine.com
+
+Configuration:
+
+```vim
+call deoplete#custom#var('tabnine', {
+\ 'line_limit': 500,
+\ 'max_num_results': 20,
+\ })
```

Comment:
Could you document what the options do? line_limit isn't a super obvious name.

Reply:
OK.  I will add the documentation.

Reply:
I have added it.

Reply:
Can you format it so it displays nicely in markdown?

Reply:
I have improved the documentation.

Reply:
Thanks!